docs/css/scroll-snap-stop.md

```diff
@@ -0,0 +1,18 @@
+> Interactivity
+
+# Scroll Snap Stop
+
+Utilities for controlling whether you can skip past possible snap positions.
+
+::: danger Unsupported
+This functionality is not yet supported! If you need this, reach out to us on [#nmp-warp-design-system](https://sch-chat.slack.com/archives/C04P0GYTHPV).
+:::
+
+## Quick reference
+
+<qr-table />
```

Comment:
Some unsupported utility classes are formatted with a line through text, and some aren't, but in the [css-docs](https://warp-ds.github.io/css-docs/scroll-snap-stop) all have such formatting.
![Screenshot 2024-06-10 at 14 38 35](https://github.com/warp-ds/tech-docs/assets/41303231/1aa31f31-1d52-4437-83f0-4413dbbd4136)

Reply:
Fixed.